renderer/components/ManualInputForm.tsx

```diff
@@ -14,10 +14,11 @@ export default function ManualInputForm({
   onSave,
 }: ManualInputFormProps) {
   const [report, setReport] = useState("");
+  const [saveBtnStatus, setSaveBtnStatus] = useState("enabled");
```

Comment:
may be use status "disabled" when our app load first time? I don't have any changes to save in this case.